doc/apiv3-documentation.apib

```diff
@@ -2271,6 +2271,397 @@ Lists the columns available for work packages of this project in the form of fie
+| delay            | The delay in days between closing of `from` and start of `to` | Integer | x >= 0                                                                                                        | READ / WRITE         |
+
+## Relation [/api/v3/relations/{id}]
+
++ Model
+    + Body
+
+            {
+                "_links":
+                {
+                    "self":
+                    {
+                        "href": "/api/v3/relations/1"
+                    },
+                    "update":
+                    {
+                      "href": "/api/v3/relations/1",
+                      "method": "PATCH"
+                    },
+                    "delete":
+                    {
+                      "href": "/api/v3/relations/1",
+                      "method": "DELETE"
+                    },
+                    "from":
+                    {
+                        "href": "/api/v3/work_packages/42",
+                        "title": "Steel Delivery"
+                    },
+                    "to":
+                    {
+                        "href": "/api/v3/work_packages/84",
+                        "title": "Bending the steel"
+                    }
+                },
+                "_type": "Relation",
+                "id": 1,
+                "type": "precedes",
+                "reverseType": "follows",
+                "description": "We can't bend the steel before it's been delivered!",
+                "delay": 0
+            }
+
+## View Relation [GET]
+
++ Parameters
+
+    + id (required, integer, `1`) ... Relation id
+
++ Response 200 (application/hal+json)
+
+    [Relation][]
+
++ Response 404 (application/hal+json)
+
+    Returned if the relation does not exist or the client does not have sufficient permissions to see it.
+
+    **Required permission:** view work packages for the involved work packages
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:NotFound",
+                "message": "The specified relation does not exist."
+            }
+
+## Edit Relation [PATCH]
+
+When calling this endpoint the client provides a single object, containing the properties and links that it wants to change, in the body.
+It is only allowed to provide properties or links supporting the **write** operation.
+
+Note that changing the `type` of a relation invariably also changes the respective `reverseType` of it.
+The returned Relation object will reflect that change. For instance if you change a Relation's
+`type` to "follows" then the `reverseType` will be changed to `precedes`.
+
++ Parameters
+
+    + id (required, integer, `1`) ... Relation ID
+
++ Request Update Relation (application/json)
+
+    + Body
+
+        {
+            "type": "blocks",
+            "description": "Actually the supplier has to bend the steel before they can deliver it."
+            "delay": 3
```

Comment:
aglio complains about this block:

`message-body asset is expected to be a pre-formatted code block, every of its line indented by exactly 12 spaces or 3 tabs (warning code 10)`